lib/install/config/loaders/installers/elm.js

```diff
@@ -1,5 +1,9 @@
+const path = require('path')
+
+const elmSource = path.join(__dirname, '..', '..', '..')
```

Comment:
[Fixed](https://github.com/bkuhlmann/webpacker/blob/elm-webpack-loader/lib/install/config/loaders/installers/elm.js#L3) (rebase). No worries, happy to explain. This is good feedback and your solution above is much better.